my-project/templates/product_category/addImages.html.twig

```diff
@@ -0,0 +1,13 @@
+{#/**#}
+{#* VI-36#}
+{#*#}
+{#* @category   Virtua#}
```

Comment:
@mSobonVirtua update all headers with module name and description in category, like CategoryGallery

Reply:
Something like that ?
{#/**#}
{#* VI-36 AddImage#}
{#*#}
{#* @category   Add image to category#}
{#* @package    AddImage_Module#}
{#* @copyright  Copyright (c) Virtua#}
{#* @author     Name <m.sobon@wearevirtua.com>#}
{#*/#}

Reply:
{#/**#}
{#* VI-36 AddCategoryImage#}
{##}
{# @category CategoryImages#}
{#* @Package Virtua_AddCategoryImage#}
{#* @copyright Copyright (c) Virtua#}
{#* @author Mateusz Sobon m.sobon@wearevirtua.com#}
{#*/#}